tests/sls/homu/valid_builders.py

```diff
@@ -0,0 +1,62 @@
+import json
+import os.path
+import subprocess
+import toml
+
+from tests.util import Failure, Success
+
+
+def are_builders_valid(buildbot, homu_cfg, buildertype):
+    for builder in homu_cfg[buildertype]:
+        if builder not in buildbot:
```

Comment:
The set differencing subsumes the need to loop over each builder and check if it is present or not, so these first two lines aren't needed.